### PR TITLE
Releasing RPi4 support to main.

### PR DESCRIPTION
**Pull request by nihemstr**
- Adds RPi4 support to the ADU proof of concept build

### CODE REVIEW

File: recipes-support/swupdate/swupdate_%.bbappend [hunk 1]
```diff
@@ -5,4 +5,5 @@ FILESEXTRAPATHS:append := "${THISDIR}/${PN}:"
 
 PACKAGECONFIG_CONFARGS = ""
 
+
```
Comment (jw-msft) [inline recipes-support/swupdate/swupdate_%.bbappend]: nit: spurious newline -- remove this change from the change set?

**Review by jw-msft — APPROVED** [review]
Approved w/ comment

File: recipes-support/swupdate/swupdate_%.bbappend (hunk 1 above)
Comment (nihemstr) [reply recipes-support/swupdate/swupdate_%.bbappend]: Addressed. Please re-review when able.

File: recipes-core/base-files/base-files/raspberrypi4-64/fstab [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+# Custom fstab entries for ADU raspberrypi3.
+# NOTE: these entries must be kept in sync with the corresponding .wks file.
+
+# Mount the boot partition that contains the bootloader.
+/dev/mmcblk0p1  /boot   vfat    defaults,sync   0   0
+
+# Mount the ADU specific partition for reading configuration and writing logs.
+# This partition is only accessible by adu user/group.
+/dev/mmcblk0p4  /adu    vfat    umask=0027,gid=800,uid=800   0   0
```
Comment (Nox-MSFT) [inline recipes-core/base-files/base-files/raspberrypi4-64/fstab:22]: If haven't done so, we should consider adding some guidance/info related to how to create 'adu' user and group, and why we chose 800 for IDs.

**Review by Nox-MSFT — APPROVED** [review]
Approved w/ comment.

**Pull request merged**